src/overlays/actors/ovl_Obj_Tokeidai/z_obj_tokeidai.c

```diff
@@ -377,15 +377,15 @@ void ObjTokeidai_TowerClock_SlideOff(ObjTokeidai* this, PlayState* play) {
 
     if (this->aerialClockFaceSpeed < 0x80) {
         thisx->shape.rot.x = this->slidingClockFaceAngle - 0x4000;
-        this->fallingClockFaceRotationalVelocity = 0x28;
+        this->fallingclockFaceAngularVelocity = 0x28;
```

Comment:
`fallingclock` -> `fallingClock` I assume this wasn't intentional?

Reply:
Whoops, let me fix this one. It's supposed to be how it was before.